routes/user.js

```diff
@@ -52,44 +38,37 @@ async function userLogin(req, res){
     const userName = req.body.userName; 
     const password = req.body.password;
     const email = req.body.email; 
-    signInWithEmailAndPassword(auth, email, password)
-    .then((userCridential) => {
-        //Signed in
-        const user = userCridential.user;
-        res.send(user);
-    })
-    .catch((error) => {
-        const errorCode = error.code; 
-        const errorMessage = error.message; 
-        console.log("Error Code: ", errorCode);
-        console.log("Error Message: ", errorMessage);
+    try {
+        const firebaseRep = await signInWithEmailAndPassword(auth, email, password);
+        const response = {
+            uid: firebaseRep.user.uid,
+            accessToken: firebaseRep.user.accessToken,
```

Comment:
Are you sure the `accessToken` is under `user` and not under `user.stsTokenManager`?
Because looking at the response I get there's no `user.accessToken` 😃 
![image](https://user-images.githubusercontent.com/8568547/169534942-83b9b5d9-29af-44df-a586-64ae115f3be8.png)